crates/shared/src/http_solver/model.rs

```diff
@@ -422,6 +424,15 @@ pub struct SimulatedTransaction {
     pub data: Vec<u8>,
 }
 
+/// Whether or not internalizable interactions should be encoded as calldata
+#[derive(Debug, Copy, Clone, Serialize)]
+pub enum InternalizationStrategy {
+    #[serde(rename = "Disabled")]
+    EncodeAllInteractions,
+    #[serde(rename = "Enabled")]
+    SkipInternalizableInteraction,
```

Comment:
Why the renames?

Reply:
I thought Enabled/Disabled is clearer to solvers, while full versions make the code more readable (at the call sites)